packages/EasyCore/src/Bridge/Symfony/Validator/Constraints/DecimalValidator.php

```diff
@@ -28,7 +28,15 @@ public function validate($value, Constraint $constraint): void
 
         $value = (string) $value;
 
-        $pattern = \sprintf('/^\d+(\.\d{%d,%d})?$/', $constraint->minPrecision, $constraint->maxPrecision);
+        if (\is_numeric($value) === true && \str_contains($value, 'E') === true) {
+            $value = \sprintf('%0.10f', $value);
+        }
+
+        if ($value !== '0') {
```

Comment:
What if `1000` is passed?
Shouldn't this `if` be inside the previous?

Reply:
If `1000` will be passed it will be ok too.

No we need it here, Ex: if '100.0000' will be passed, we need to remove trailing zeros.

Reply:
If we cut trailing zeros from any integer it still will be accepted by regexp.

Reply:
OK, got it, thanks